src/helpers/quantityPurchase.js

```diff
@@ -0,0 +1,11 @@
+function quantityPurchase (tvArray) {
+    let total = 0;
+
+    for (let i = 0; i < tvArray.length; i++) {
+        total = total + tvArray[i].originalStock;
+    }
+
+    return total;
+}
+
+export default quantityPurchase;
```

Comment:
Hierbij stond ook in de opdracht dat je het moest loggen in de console;
console.log("Totaal aantal ingekochte tv's:", amountBought(inventory));

Maar ziet er ook netjes uit.
Je kan de total = total + tvArray[i].originalStock; korter opschrijven door dit te gebruiken;
total += tvArray[i].originalStock;